src/Ting/Repository/Repository.php

```diff
@@ -289,10 +289,12 @@ public function getAll($forceMaster = false)
      * @param bool  $forceMaster
      * @return CollectionInterface<T>
      */
-    public function getBy(array $criteria, $forceMaster = false)
+    public function getBy(array $criteria, $forceMaster = false, $order = [], $limit = 0)
```

Comment:
can you add the relevant type hints in the signature ?

Reply:
I added typehint for parameters `$order` and `$limit`